### ISSUE TITLE
(CLOSED) Docker failed to start: Unable to stop: Couldn't stop VM MobyLinuxVM

### ISSUE THREAD

**Issue by awidjaja** [docker/for-win#1119]
### Expected behavior
Docker is running

### Actual behavior
```
Unable to stop: Couldn't stop VM MobyLinuxVM
at Fatal, <No file>: line 376
at Stop-VM-Force, <No file>: line 372
at Stop-MobyLinuxVM, <No file>: line 302
at <ScriptBlock>, <No file>: line 383
   at Docker.Backend.HyperV.RunScript(String action, Dictionary`2 parameters)
   at Docker.Backend.ContainerEngine.Linux.DoStop()
   at Docker.Backend.ContainerEngine.Linux.Start(Settings settings)
   at Docker.Core.Pipe.NamedPipeServer.<>c__DisplayClass9_0.<Register>b__0(Object[] parameters)
   at Docker.Core.Pipe.NamedPipeServer.RunAction(String action, Object[] parameters)
```
### Information
  - Diagnostic ID from "Diagnose & Feedback" in the menu.
649D497D-1150-4487-906C-5EB11CBB6907/2017-09-23_11-00-47
  - a reproducible case if this is a bug, Dockerfiles FTW
N/A
  - page URL if this is a docs issue or the name of a man page
N/A
  - host distribution and version (Windows version, build number, etc)
```
Edition: Windows 10 Pro
Version: 1703
OS Build: 15063.608
Processor: Intel(R) Core(TM) i7-3687U CPU @2.10 GHz 2.10 GHz
Installed RAM: 4.00 GB (3.88 GB usable)
System type: 64-bit OS, x64-based processor
Socket: 1
Cores: 2
Logical processors: 4
Virtualization: Enabled
Connected to network via Wi-Fi
Adapters:
  Adapter name: Wi-Fi
  IPv4 address: 192.168.1.4
  Adapter name: vEthernet (HNS Internal NIC)
  IPv4 address: 172.31.128.1
```

### Steps to reproduce the behavior

  1. Install docker for windows on Windows 10 build 1703
  2. Restart and wait for docker services to load

Had tried without success:
1. Reset to factory default (--> same issue)
2. Forced shutdown the mobilinux VM in Hyper-V manager and restart (--> same issue)
3. Switched to Windows container (running OK), restart, switched back to Linux containers (--> same issue)
4. Uninstalled Docker for Windows and disable Hyper-V features, restart and install back Docker for Windows, click enable Hyper-V during installed (--> same issue)

Please help urgently.

[log.txt](https://github.com/docker/for-win/files/1326256/log.txt)

**Comment by awidjaja** [comment]
Variation of the error:
```
Docker hv-sock proxy (vsudd) is not reachable
   at Docker.Backend.ContainerEngine.Linux.ConnectToVsud(TaskCompletionSource`1 vmId) in C:\gopath\src\github.com\docker\pinata\win\src\Docker.Backend\ContainerEngine\Linux.cs:line 267
   at Docker.Backend.ContainerEngine.Linux.DoStart(Settings settings) in C:\gopath\src\github.com\docker\pinata\win\src\Docker.Backend\ContainerEngine\Linux.cs:line 251
   at Docker.Backend.ContainerEngine.Linux.Start(Settings settings) in C:\gopath\src\github.com\docker\pinata\win\src\Docker.Backend\ContainerEngine\Linux.cs:line 123
   at Docker.Core.Pipe.NamedPipeServer.<>c__DisplayClass9_0.<Register>b__0(Object[] parameters) in C:\gopath\src\github.com\docker\pinata\win\src\Docker.Core\pipe\NamedPipeServer.cs:line 47
   at Docker.Core.Pipe.NamedPipeServer.RunAction(String action, Object[] parameters) in C:\gopath\src\github.com\docker\pinata\win\src\Docker.Core\pipe\NamedPipeServer.cs:line 145
```
Seems to me docker for windows is still buggy and not ready for prime time... am I correct?

[log.txt](https://github.com/docker/for-win/files/1326432/log.txt)

**Comment by julianonunes** [comment]
Also having the same problem here (it was working fine yesterday).

282B02C4-13B0-4E11-BCD8-A101EC583FF8/2017-11-09_12-45-27

**Comment by awidjaja** [comment]
(Closed) No longer an issue with the Version 17.12.0-ce-rc2-win41 (14746)

**Comment by harisrinivaas** [comment]
switch to windows containers (right clink on docker icon on quick lunch) solved the problem.
